CONTRIBUTING.md

```diff
@@ -110,7 +162,7 @@ We use the following label scheme to categorize issues:
 -   **area** - `addon: x`, `addons-api`, `stories-api`, `ui`, etc.
 -   **status** - `needs reproduction`, `needs PR`, `in progress`, etc.
 
-All issues should have a `type` label. `bug`/`feature`/`question`/`discussion` are self-explanatory. `greenkeeper` is for keeping package dependencies up to date. `maintenance` is a catch-all for any kind of cleanup or refactoring.
+All issues should have a `type` label. `bug`/`feature`/`question`/`discussion` are self-explanatory. `greenkeeper` is for keeping package dependencies up to date. `maintenance` is ta catch-all for any kind of cleanup or refactoring.
```

Comment:
I think the current tag for dependencies is just dependencies. Also maintenance is -a- catch all